The sudo: tag, Trusty, and Python 3.4 are all End of Life

cclauss opened the pull request:
@ucalyptus

cclauss committed:
The sudo: tag, Trusty, and Python 3.4 are all End of Life

@ucalyptus

ucalyptus commented on .travis.yml:
```diff
@@ -1,15 +1,14 @@
 
 env:
-    matrix:
-        - TASK=lint
```
Wasn't aware that this goes here without the matrix.

cclauss replied:
(`matrix` is EOL in favour of `jobs`)

ucalyptus commented:
something wrong with the __init__.py

cclauss committed:
pylint skflow

cclauss commented:
`skflow/__init__.py`:24:0: C0305: Trailing newlines (trailing-newlines)

cclauss committed:
Remove a extra \n ?

cclauss committed:
travis_install.sh: Python 3.4 --> 3.5

cclauss committed:
Travis CI: Python 3.8 --> 3.5

https://docs.travis-ci.com/user/languages/python/#travis-ci-uses-isolated-virtualenvs

cclauss commented:
We have to use Python 3.5 instead of 3.8 because of
https://docs.travis-ci.com/user/languages/python/#travis-ci-uses-isolated-virtualenvs

ucalyptus commented on scripts/travis_install.sh:
```diff
@@ -41,8 +41,8 @@ case "$TASK" in
-                    "3.4")
-                        TENSORFLOW_PACKAGE_URL="https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.8.0rc0-cp34-cp34m-linux_x86_64.whl"
+                    "3.5")
+                        TENSORFLOW_PACKAGE_URL="https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.8.0rc0-cp35-cp35m-linux_x86_64.whl"
```
@cclauss This is causing a Missing wheel error at workflow build

ucalyptus replied:
https://storage.googleapis.com/tensorflow/linux/cpu/tensorflow-0.8.0rc0-cp35-cp35m-linux_x86_64.whl does not exist

ucalyptus reviewed (changes requested):
Find a version that works for 3.5

ucalyptus commented:
Check #191

cclauss commented:
Closing in favor of #191
